test cases/common/227 fs module/meson.build

```diff
@@ -18,4 +20,53 @@ assert(fs.is_dir('subprojects'), 'Dir not detected correctly.')
 assert(not fs.is_dir('meson.build'), 'File detected as a dir.')
 assert(not fs.is_dir('nonexisting'), 'Bad path detected as a dir.')
 
+assert(fs.is_dir('~'), 'expanduser not working')
+assert(not fs.is_file('~'), 'expanduser not working')
+
+original = 'foo.txt'
+new = fs.replace_suffix(original, '.ini')
+assert(new.endswith('foo.ini') and not new.contains('.txt'), 'replace_suffix failed')
```

Comment:
Why this complicated form of comparison? Why not just `assert(new == 'foo.ini', ...`? And the same for the other test asserts.